### PR TITLE
Update "Cart Integrations" section of guide

### PR DESCRIPTION
**Pull request by ScottRudiger**
Changes:

- Update section title "Cart Integrations" => "API Integrations"
- Emphasize that official integrations are built in-house
- Remove Stripe
- Add NetSuite
- Add paragraph on certified integrations

### CODE REVIEW

**Comment by ScottRudiger** [comment]
@fastdivision, should we add paragraphs for NetSuite and WooCommerce below Magento?

File: source/guides.md [hunk 1]
```diff
@@ -309,3 +309,7 @@ Our Magento sales tax extensions currently support Magento 1.7.x - 1.9.x and Mag
 - Install our [Magento 2](https://marketplace.magento.com/taxjar-module-taxjar.html) or [Magento 1](https://marketplace.magento.com/taxjar-taxjar-salestaxautomation.html) extension from Magento Marketplace
 - Get started and learn how it works with our [Extension Guide](https://www.taxjar.com/guides/integrations/magento/)
 - Browse the code in our [GitHub Repository](https://github.com/taxjar/taxjar-magento-extension)
+
+## Certified Integrations
+
+Dozens of certified integrations are available for other popular eCommerce platforms as well. These integrations are built and maintained by third-party developers and tested and certified by the TaxJar team. See the [full list of integrations built by our partners](https://www.taxjar.com/sales-tax-integrations/).
```
Comment (fastdivision) [inline source/guides.md]: Consider revising the first sentence here to cover eCommerce, ERP, POS, accounting, inventory management, order management, marketplaces, and other platforms.

**Comment by fastdivision** [comment]
@ScottRudiger Yes let's add sections for each official integration. Thank you!

**Comment by ScottRudiger** [comment]
@fastdivision Let me know if there's anything else we should add to each official integration's section.

File: source/guides.md [hunk 2]
```diff
@@ -309,3 +309,21 @@ Our Magento sales tax extensions currently support Magento 1.7.x - 1.9.x and Mag
...
+Our NetSuite integration requires a [TaxJar Plus](https://www.taxjar.com/plus/) subscription and your NetSuite account must have SuiteTax enabled.
+
+- Learn how to install and configure the integration with our [NetSuite Guide](https://www.taxjar.com/guides/integrations/netsuite/)
+
+## WooCommerce
+
+Our WooCommerce plugin is compatible with WooCommerce 2.6+ and WordPress 4.2+.
+
+- Install our [WooCommerce](https://wordpress.org/plugins/taxjar-simplified-taxes-for-woocommerce/#installation) plugin from the WordPress Plugin Directory
+- Learn more or get started right away with our [Guide for WooCommerce Sellers](https://www.taxjar.com/guides/sales-tax-guide-for-woocommerce/)
```
Comment (fastdivision) [inline source/guides.md]: @ScottRudiger Let's link to this documentation instead: https://docs.woocommerce.com/document/taxjar/

Comment (ScottRudiger) [reply source/guides.md]: Done 👍

**Pull request merged**